complement errors relating to lazy uniqueness check (#10928)
pingcap/docs: constraints.md

    ```

    ```
    ERROR 8147 (23000): transaction aborted because lazy uniqueness check is enabled and an error occurred: [kv:1062]Duplicate entry 'bill' for key 'users.username'
    ```

> - When this variable is disabled, the `1062 Duplicate entry` error does not indicates that it occurs in the current SQL statement. Therefore, when a transaction operates on multiple tables that have indexes with the same name, please check the `1062` error message to find which actual index the error is from.

Reviewer: ```suggestion
- When this variable is disabled, the `1062 Duplicate entry` error might be not from the current SQL statement. Therefore, when a transaction operates on multiple tables that have indexes with the same name, you need to check the `1062` error message to find which index the error is actually from.
```

> - When this variable is disabled, the `1062 Duplicate entry` error does not necessarily happen in the currently executing SQL statement. Therefore, when a transaction operates on multiple tables that have indexes with the same name, please check the `1062` error message to find which actual index the error is from.

Reviewer 2: ```suggestion
- When this variable is disabled, the `1062 Duplicate entry` error does not indicates that it occurs in the current SQL statement. Therefore, when a transaction operates on multiple tables that have indexes with the same name, please check the `1062` error message to find which actual index the error is from.
```

> - When this variable is disabled, the `1062` Duplicate entry error does not necessarily happen in the currently executing SQL statement. Therefore, when a transaction operates on multiple tables that have indexes with the same name, please check the `1062` error message to find which actual index the error is from .

Author: ```suggestion
- When this variable is disabled, the `1062 Duplicate entry` error does not necessarily happen in the currently executing SQL statement. Therefore, when a transaction operates on multiple tables that have indexes with the same name, please check the `1062` error message to find which actual index the error is from .
```

> - When this variable is disabled, the `1062 Duplicate entry` error does not necessarily happen in the currently executing SQL statement. Therefore, when a transaction operates on multiple tables that have indexes with the same name, please check the `1062` error message to find which actual index the error is from .

Reviewer 3: ```suggestion
- When this variable is disabled, the `1062 Duplicate entry` error does not necessarily happen in the currently executing SQL statement. Therefore, when a transaction operates on multiple tables that have indexes with the same name, please check the `1062` error message to find which actual index the error is from.
```
